src/game/game.rs

```diff
@@ -6,21 +6,21 @@ use wasm_bindgen::prelude::*;
 #[wasm_bindgen]
 #[derive(Debug)]
 pub struct Game {
-    _num_decks: u8,
-    _num_jokers: u8,
+    num_decks: u8,
+    num_jokers: u8,
     players: Vec<Player>,
```

Comment:
I don't think game needs players, maybe player ids is enough since players are not used in the game, and we can use the round get_player function.